models/edr/run_results/test_results.sql

```diff
@@ -0,0 +1,49 @@
+{{
```

Comment:
Choose a different name for this model that conveys on the fact that it only applies to currently existing tests (like "latest", "current") and use that name throughout.